src/test/java/org/broadinstitute/hellbender/engine/ReadsDataSourceUnitTest.java

```diff
@@ -573,7 +573,7 @@ public void testManuallySpecifiedIndices( final List<Path> bams, final List<Path
     public void testManuallySpecifiedIndicesWithCustomReaderFactory( final List<Path> bams, final List<Path> indices ) {
         final SamReaderFactory customFactory = SamReaderFactory.makeDefault().validationStringency(ValidationStringency.STRICT);
 
-        try ( final ReadsDataSource readsSource = new ReadsDataSource(bams, indices, customFactory) ) {
+        try ( final ReadsDataSource readsSource = new ReadsDataSource(bams, indices, customFactory, null, null) ) {
```

Comment:
To be thorough I have also added a test with files on the Google Cloud (I've added them both there and on the repo) where the files are XOR'd with a constant, and the wrapper undoes this. This detects if we ignore the wrapper or use the wrong one.